WTWSMS/events/WtWSMS_enlil_religious.txt

```diff
@@ -1292,25 +1289,46 @@ character_event = {
 	}
 }
 
-# Shown when a Melchisidechian creates the Kingdom of Salem
+#Shown when a Melchisedechian creates the Kingdom of Salem
 narrative_event = { 
 	id = spirit.40
 	title = spirit.40.name
 	desc = spirit.40.desc
 	picture = GFX_evt_gnostics_rising
 	border = GFX_event_narrative_frame_learning
 	
+	major = yes
+
```

Comment:
Why? "MAJOR = YES" makes everony see it. That's the point of it being a Narrative Event!